docs/tutorials/add-contracts-pallet.md

```diff
@@ -448,12 +442,12 @@ jsonrpc-core = '14.0.5'
 
 [dependencies.pallet-contracts-rpc]
 git = 'https://github.com/paritytech/substrate.git'
-version = '0.8.0-rc4'
-tag = 'v2.0.0-rc4'
+version = '0.8.0-rc6'
+tag = 'v2.0.0-rc6'
 
 [dependencies.sc-rpc]
 git = 'https://github.com/paritytech/substrate.git'
-tag = 'v2.0.0-rc4'
+tag = 'v2.0.0-rc6'
 ```
 
 **`node/src/service.rs`**
```

Comment:
Line number 471, .with_rpc_extensions is can not used in macro_rules! new_full_start